tests/lax_numpy_test.py

```diff
@@ -194,6 +194,8 @@ def op_record(name, nargs, dtypes, shapes, rng_factory, diff_modes,
               jtu.rand_nonzero, ["rev"]),
     op_record("floor_divide", 2, uint_dtypes, all_shapes,
               jtu.rand_nonzero, ["rev"]),
+    op_record("fmin", 2, float_dtypes, all_shapes, jtu.rand_some_nan, []),
```

Comment:
Shouldn't this support and test complex types too? (I mention this because the NumPy docs explicitly give semantics to the complex NaN case if nothing else.)

```
In [1]: import numpy as np

In [2]: np.fmin(2+3j, 4+5j)
Out[2]: (2+3j)
```

Reply:
Good point! Done.